libs/sdk-ui-tests-e2e/scenarios/src/index.tsx

```diff
@@ -1,16 +1,17 @@
-// (C) 2022 GoodData Corporation
+// (C) 2022-2023 GoodData Corporation
 import "react-app-polyfill/stable";
 import React from "react";
 import ReactDOM from "react-dom";
 
 import App from "./App";
 import { AppProviders } from "./contexts";
 
-import "@gooddata/sdk-ui-charts/styles/css/main.css";
-import "@gooddata/sdk-ui-ext/styles/css/main.css";
 import "@gooddata/sdk-ui-filters/styles/css/main.css";
-import "@gooddata/sdk-ui-geo/styles/css/main.css";
+import "@gooddata/sdk-ui-charts/styles/css/main.css";
 import "@gooddata/sdk-ui-pivot/styles/css/main.css";
+import "@gooddata/sdk-ui-geo/styles/css/main.css";
+import "@gooddata/sdk-ui-kit/styles/css/main.css";
```

Comment:
why do you import sdk-ui-kit?

Reply:
our app is missing this css, it causes broken some parts in the dashboard